### PR TITLE
Automate website deployment with travis-ci

### PR DESCRIPTION
**Pull request by ghost**
The website needs automatic deployments as the current process is to time intensive for the staff and makes it essentially impossible to deliver quick fixes to the website.

The deploy script uses curl to copy the new changes on the master branch on to the server which hosts [innovation.fsu.edu](https://innovation.fsu.edu)

### CODE REVIEW

**Comment by kjbaldauf** [comment]
Thanks Gwen. Let's work to fix this prior to the end of the semester. I probably won't be able to give it any time until after the 20th.


Best Regards,

Ken Baldauf

__________________________________

Director, FSU Innovation Hub

www.innovation.fsu.edu<http://www.innovation.fsu.edu/>

❤️💡🚀


________________________________

**Comment by ghost** [comment]
@kjbaldauf  I'm actually planning to merge this myself tomorrow. Thanks for being vigilant though!

**Comment by ghost** [comment]
@FabuolaP, I added deploy previews with [netlify](https://netlify.com), this allows us to do cool things like see what our changes _will_ look like when they are pushed.

deploy

If you click the [`Details`](https://deploy-preview-22--silly-dubinsky-f9015b.netlify.com/) button, you will see the deploy preview for the site so you and @kjbaldauf can see how your changes will turn out!  The site is _very_ large, however, so the deploy previews may take a few minutes to pop up.

File: .deploy.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+if [ $# -lt 3 ]; then
+    echo "ERROR: missing arguments"
+    usage
+    exit 2
+fi
+
+echo "Updating Files"
+
+curl --user "$1:$2" -T . "$3"
```
Comment (kjbaldauf) [inline .deploy.sh:15]: [Upload to dev folder in WebDAV to test](https://faq.its.fsu.edu/web-services/webdav-departments)

**Pull request closed without merging**